qutebrowser/browser/commands.py

```diff
@@ -282,7 +282,12 @@ def tab_pin(self, count=None):
         to_pin = not tab.data.pinned
         tab_index = self._current_index() if count is None else count - 1
         cmdutils.check_overflow(tab_index + 1, 'int')
-        self._tabbed_browser.set_tab_pinned(tab_index, to_pin)
+        tab = self._cntwidget(count)
+
+        if tab is None:
+            raise cmdexc.CommandError("Unable to find tab '{}'.".format(count))
```

Comment:
Ah, I probably should have seen all these things, sorry :P.

I probably need to get more sleep....